new/client/src/components/auth/Auth.js

```diff
@@ -79,8 +139,13 @@ class AuthPage extends Component {
   };
 
   render() {
+    if(this.state.isLogin){
     return (
-      <form className="auth-form" onSubmit={this.submitHandler}>
+      <form className="auth-form" onSubmit={this.submitLogin}>
+        <div className="form-control">
+          <label htmlFor="username">Username</label>
+          <input type="username" id="username" ref={this.userNameEl} />
+        </div>
         <div className="form-control">
```

Comment:
The user need not enter both username and email while logging in. 
The input  field should accept both email and username and log him in, like it is for instagram for example.